### DISCUSSION TITLE
Task request: NeRF in the Wild?

### DISCUSSION THREAD

**Discussion by QROST** [NVlabs/instant-ngp#149] [Ideas / Feature requests]
Absolutely brilliant work! 
Greetings from my cat Popcorn, NeRF from only **25** dirty images, with 3080 under 5 minutes: [instagram](https://www.instagram.com/tv/CY3KKuTo1aP/?utm_source=ig_web_copy_link )

It would be superb if this instant-ngp could reach **NeRF in the Wild** level of fidelity using unconstrained large datasets... e.g.: [Image Matching Challenge PhotoTourism (IMC-PT)](https://www.cs.ubc.ca/~kmyi/imw2020/data.html)

Any chance? 
[NeRF in the Wild](https://nerf-w.github.io/)

**Comment by QROST** [comment]
one NeRF-W implementation we could reference: [nerfw](https://github.com/kwea123/nerf_pl/tree/nerfw)

**Comment by ookey** [comment]
It would be great, indeed! 
I'd better get the inspiration from [ADOP](https://github.com/darglein/ADOP) paper, which management of white balance and other cam specific params seems hyper clever!

**Comment by kwea123** [comment]
I'm the author of nerfw repo, and recently just wrote a pytorch version of ngp https://github.com/kwea123/ngp_pl
It might be simpler to pick the components from nerfw and integrate into ngp_pl. I am focusing on other tasks now, but feel free to discuss if you are attempting!
